src/main/java/org/folio/innreach/controller/InnReachTransactionController.java

```diff
@@ -60,4 +61,11 @@
     var transactions = transactionService.getAllTransactions(offset, limit, parameters);
     return ResponseEntity.ok(transactions);
   }
+
+  @Override
+  @PutMapping("/inn-reach/transactions/update/{id}")
```

Comment:
PUT method usually means "update" operation. What was the reason behind adding "/update/" to URI?

Reply:
removed